device-types/HPE/Aruba-2920-48G-PoEP-740W.yaml

```diff
@@ -0,0 +1,125 @@
+---
+manufacturer: HPE
+model: Aruba 2920-48G-PoE+-740W
+slug: hpe-aruba-2920-48g-poep-740w
+part_number: J9836A
+u_height: 1
+is_full_depth: false
+weight: 5830
+weight_unit: g
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 881
+    allocated_draw: 52
```

Comment:
It appears that the PSU is hot swappable. Thus it should be modelled under `module-bays`, and then the associated PSU module should be modelled in the module-types folder if they do not already exist there

If I am wrong about this please let me know and mark this as resolved